cmd/internal/pipeline/evaluator.go

```diff
@@ -0,0 +1,59 @@
+package pipeline
+
+import (
+	"context"
+	"fmt"
+	"github.com/hacbs-contract/ec-cli/cmd/internal/utils"
+	"github.com/open-policy-agent/conftest/runner"
+	"github.com/spf13/afero"
+)
+
+var createWorkDir = afero.TempDir
+
+type Evaluator struct {
+	Context       context.Context
+	Target        EvaluationTarget
+	PolicySources []PolicySource
+	Paths         ConfigurationPaths
+	TestRunner    runner.TestRunner
+	Namespace     []string
+	OutputFormat  string
+	workDir       string
+}
+
+func (e *Evaluator) addDataPath() error {
+	dataDir := fmt.Sprintf("%s/data/", e.workDir)
+	exists, err := afero.DirExists(utils.AppFS, dataDir)
+	if err != nil {
+		return err
+	}
+	if !exists {
+		_ = utils.AppFS.MkdirAll(dataDir, 0755)
+		err = afero.WriteFile(utils.AppFS, fmt.Sprintf("%s/data/data.json", e.workDir), []byte("{\"config\":{}}\n"), 0777)
```

Comment:
Should the config be in `data/config/policy/data.json`? We could use `filepath.Join` here as well

Reply:
That's a good question, do we have any documentation on where we expect it to exist? In the [original bash script](https://github.com/redhat-appstudio/build-definitions/blob/0b05a990917f20f82394d02f28841500df23e612/hack/verify-pipeline.sh#L26), `DATA_DIR` was defined as `$EC_WORK_DIR/data`, which I've replicated here.

Reply:
I think what we have here is correct at least according to the [current Makefile](https://github.com/hacbs-contract/ec-policies/blob/f17f2e54d59560b5b84ce5b9a1f08b59573f53de/Makefile#L186-L187), we _might_ have had it in `data/config/policy/data.json` before or I imagined it to be so.